README.md

```diff
@@ -0,0 +1,62 @@
+## Description
```

Comment:
It should mention `jeos-firstboot` somewhere, either in a title or the first sentence